sass/base/layout/_layout.scss

```diff
@@ -22,6 +22,21 @@
 .desktop-show{
   display: inherit;
 }
+/* Flex Container Direction */
+.flex-container{
+  &.desktop-ltr{
+    flex-direction: row;
+  }
+  &.desktop-rtl{
+    flex-direction: row-reverse;
+  }
+  &.desktop-ttb{
+    flex-direction: column;
+  }
+  &.desktop-btt{
+    flex-direction: column-reverse;
+  }
+}
```

Comment:
There are still duplicate rules inside of the media query.
We could also still make use of `breakpointDesktop`, by simply moving those styles outside of the media query, **but still within the `@each`**
```scss
/*
  DESKTOP BREAKPOINTS
  Loop through each desktop defined breakpoint
*/
@each $label, $breakpoint in getThemeProperty(breakpointDesktop){

  /* Flex Container Direction */
  .flex-container{
    &.#{$label}-ltr{
      flex-direction: row;
    }
    &.#{$label}-rtl{
      flex-direction: row-reverse;
    }
    &.#{$label}-ttb{
      flex-direction: column;
    }
    &.#{$label}-btt{
      flex-direction: column-reverse;
    }
  }

  @media screen and (min-width: $breakpoint) {
    .#{$label}-half{
      width: 0.5%;
      min-height: 1px;
    }
    @for $i from 1 through 100 {
      .#{$label}-#{$i} {
        width: round(percentage(math.div($i, 100)));
      }
    }
    .#{$label}-auto{
      width: auto;
    }
    .#{$label}-hide{
      display: none;
    }
    .#{$label}-show{
      display: inherit;
    }
  }
}
```

Reply:
> There are still duplicate rules inside of the media query.

Right, not sure how to get around this. The "desktop-" rules outside of the mediaquery act as a default/fall back mostly - we could rename those to "default-" ....but that would be a major change and require more class tags to be used in the html. If we are looking to optimize this and remove the duplicate rules I think we should handle in a separate PR

> We could also still make use of breakpointDesktop, by simply moving those styles outside of the media query, but still within the @each

But this would add those for each breakpoint added to the desktop list and they would exist outside of their related mediaquery